.github/workflows/release-label-check.yml

```diff
@@ -0,0 +1,45 @@
+name: Enforce Release Label
+
+permissions:
+  actions: none
+  checks: write
+  contents: read
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: read
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+jobs:
+  check-Label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5.1.0
+        with:
+          script: |
+            const acceptableLabels = ['major','minor','patch']
+            const {data} = await github.rest.issues.listLabelsOnIssue({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const labels = data.map(label => label.name)
+            if (!data || !(labels.some(item => acceptableLabels.includes(item)))) {
```

Comment:
the check `!data` is either not required or too late - depending on what you wanted to achieve. You are already calling `data.map` in the previous line that would fail when data is not defined. 

The logic could be simplified further: 

```js
const releaseLabels = labels.filter(label => acceptableLabels.includes(label))
if (releaseLabels.lenght < 1) // No Realse Labels
if (releaseLabels.lenght > 1) // Multiple

```
Then you can get rid of the complex if-condition in line 39 completely

Reply:
that makes sense, I tested it with no labels and it worked now, last time i tried it failed.